Update tumor detection tutorial (#1370)
Project-MONAI/tutorials: pathology/tumor_detection/README.MD

The model is based on ResNet18 with the last fully connected layer replaced by a 1x1 convolution layer.

## Pre-trained weights

> A pre-trained encoder weights would benefit the model training. In this tutorial, we initialized the model with pre-trained weights on the [ImageNet dataset](https://ieeexplore.ieee.org/document/5206848). You could using `--no-pretrain` to deactivate ImageNet weights.

Reviewer: ```suggestion
A pre-trained encoder weights would be beneficial for model training. In this tutorial, we initialized the model with pre-trained weights on the [ImageNet dataset](https://ieeexplore.ieee.org/document/5206848). You can use `--no-pretrain` to deactivate ImageNet weights.
```

Author: Have updated it, thanks!
